project-koku/koku
# [COST-4028] drop OCPPodSummaryByNodeP table

## Jira Ticket

[COST-4028](https://issues.redhat.com/browse/COST-4028)

## Description

The `OCPPodSummaryByNodeP` table was not created as a partitioned table. This PR will drop that table, then it can be recreated, correctly, as a partitioned table. The SQL to populate this table has also been commented out.

## Testing

1. run migration
2. see that `reporting_ocp_pod_summary_by_node_p` has been removed.

## Notes

...
